website/content/docs/plugins/creation/index.mdx

```diff
@@ -212,89 +167,64 @@ Here's what you need to create releases using GitHub Actions:
-
-The registration process requires the creation of a `docs.zip` file archive containing the `.mdx` files for each of the plugin components in the remote plugin's repository. A working example can be seen at the [packer-plugin-docker repository](https://github.com/hashicorp/packer-plugin-docker/releases/latest).
-
-Once in place the remote plugin can be added to Packer's website builds by opening a pull-request against [hashicorp/packer](https://github.com/hashicorp/packer), with the needed configuration for pulling in the remote documentation.
-
-Remote plugins will have their components listed under the respected types (i.e builders, provisioners, etc) using the names specified in the remote block configuration, and labeled with their respective [tier and namespace](/packer/docs/plugins#tiers-and-namespaces).
-
-To register a plugin follow one of the following setups
-
-<Tabs>
-<Tab heading="Generating docs.zip on plugin release">
-
-Documentation for a plugin is maintained within the `docs` directory and served on GitHub.
-
-To include plugin docs on the website, a global pre-hook has been added to the main scaffolding [.goreleaser.yml](https://github.com/hashicorp/packer-plugin-scaffolding/blob/42e5b0b1e575879b0477cb6d4291e027f4d92f85/.goreleaser.yml#L10) file, that if uncommented will generate and include a docs.zip file as part of the plugin release.
-
-The `docs.zip` file contains all of the `.mdx` files under the plugins root `docs/` directory that the website can consume remotely.
-
-</Tab>
-
-<Tab heading="Manually generating docs.zip">
-
-You can generate the required documentation structure manually by creating a zip file called `docs.zip` of the docs directory and including that zip file in the plugin release.
-
-```/bin/bash
-[[ -d docs/ ]] && zip -r docs.zip docs/
+   [init docs](https://developer.hashicorp.com/packer/docs/commands/init).
+
+## Registering Plugins
+
+~> Note: Registering a plugin as an integration requires the use of [Packer's Incofiguration configuration](https://github.com/hashicorp/packer-plugin-scaffolding/tree/main/docs).
+
+To help with the discovery of Packer plugins, plugins maintainers can choose to register their plugin as a [Packer Integration](https://developer.hashicorp.com/packer/integrations).
+
+The registration process requires [metadata configuration](https://github.com/hashicorp/integration-template#metadata-configuration) be added to your plugin repository for configuring the Packer integration pipeline and 
+a specific directory structure for plugin documentation to be render on the [Packer Integrations](https://developer.hashicorp.com/packer/integrations) portal.
+
+You can execute the following steps to register your plugin as an integration:
+
+1. Update your plugin documentation structure according to the template defined with the [Packer Plugin Scaffolding template].
```

Comment:
```suggestion
1. Update your plugin documentation structure according to the template defined with the [Packer Plugin Scaffolding template](https://github.com/hashicorp/packer-plugin-scaffolding/tree/main/docs).
```

Reply:
@lbajolet-hashicorp I updated the suggestion. Let me know what you think?

Reply:
I'm not too sure about the `Plugins using `packer-sdc` for generating documentation partials should look to create a new directory for serving the fully rendered documentation.`
`packer-sdc` is built-in and required to generate the hcl2spec Go files for the structures exchanged over-the-wire, so I think it's reasonable to assume devs are using it.
I understand they may not rely on it for documentation purposes, but that's minor I'd think.

Reply:
Okay I'll remove that wording